extensions/gitlab/src/gitlabapi.ts

```diff
@@ -339,9 +343,8 @@ export class GitLab {
   }
 
   public async fetch(url: string, params: { [key: string]: string } = {}, all = false): Promise<any> {
-    const per_page = all ? 100 : 50;
     const fetchPage = async (page: number): Promise<Response> => {
-      const pagedParams = { ...params, ...{ per_page: `${per_page}`, page: `${page}` } };
+      const pagedParams = { ...params, ...{ per_page: "50", page: `${page}` } };
```

Comment:
Why limit to 50? The [GitLab docs](https://docs.gitlab.com/ee/api/index.html#offset-based-pagination) mention that 100 is max, so with 50 this will double the requests in some cases.

Reply:
No particular reason. I was thinking that 100 page size was a "hack" to fake pagination. 

But indeed we can keep 50 when fetching a single page, and 100 when fetching them all to limit requests.

Reply:
Please a it back to reduce requests for larger data volumes.